### ISSUE TITLE
k8s.io/kubernetes/examples test failed

### ISSUE THREAD

**Issue by y2kenny** [kubernetes/kubernetes#60905]
/kind bug

**What happened**:
Test failed when using ./build/run.sh make test
```
ok     k8s.io/kubernetes/cmd/kubelet/app/options    0.038s
--- FAIL: TestExampleObjectSchemas (0.16s)
    examples_test.go:386: Directory ../examples/elasticsearch: Expected 3 examples, Got 0
    examples_test.go:356: skipping : ../examples/openshift-origin/secret.json/secret
FAIL
FAIL    k8s.io/kubernetes/examples    0.201s
ok     k8s.io/kubernetes/hack    0.003s
```

**What you expected to happen**:
Test passing

**How to reproduce it (as minimally and precisely as possible)**:
./build/run.sh make test

**Anything else we need to know?**:

**Environment**:
Checked out code at master:5364f7beb8
Ubuntu 16.04.3 LTS
kernel:  4.10.0-40-generic
$ go version
go version go1.9.4 linux/amd64
$ docker version
Client:
 Version:      17.09.0-ce
 API version:  1.32
 Go version:   go1.8.3
 Git commit:   afdb6d4
 Built:        Tue Sep 26 22:42:18 2017
 OS/Arch:      linux/amd64
Server:
 Version:      17.09.0-ce
 API version:  1.32 (minimum version 1.12)
 Go version:   go1.8.3
 Git commit:   afdb6d4
 Built:        Tue Sep 26 22:40:56 2017
 OS/Arch:      linux/amd64
 Experimental: false

**Comment by feiskyer** [comment]
Let me file a PR to fix this

**Comment by tpepper** [comment]
Given status in #60365 should this issue be also be labelled for milestone 1.10?  And is the issue fully resolved by #60365?  Since the issue's not linked there and is still open here it's unclear.

**Comment by tpepper** [comment]
@feiskyer should this be labelled for 1.10?

**Comment by feiskyer** [comment]
@tpepper Yep, I think so.

**Comment by feiskyer** [comment]
Fixed in #60365.
